src/tools/miri/tests/fail/read_from_trivial_switch.rs

```diff
@@ -0,0 +1,8 @@
+use std::mem::MaybeUninit;
```

Comment:
Please add a top-level doc comment explaining what this is testing and referencing the issue.

Reply:
This comment is still unresolved.